### PR TITLE
Tests for Format Number class

### PR DESCRIPTION
**Pull request by turbo124**

### CODE REVIEW

File: src/common/helpers/invoices/invoice-sum-inclusive.ts [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  protected async calculateDiscount() {
+    this.totalDiscount = this.discount(this.invoiceItems.subTotal);
+    this.total -= this.totalDiscount;
+
+    return this;
+  }
+
+  protected async calculateInvoiceTaxes() {
+
+    let amount = this.total;
```
Comment (beganovich) [inline src/common/helpers/invoices/invoice-sum-inclusive.ts:52]: This variable isn't used anywhere. Should we return it or update some state based on it, or?

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, slight change needed.

File: src/common/helpers/invoices/invoice-sum-inclusive.ts (hunk 1 above)
Comment (turbo124) [reply src/common/helpers/invoices/invoice-sum-inclusive.ts:52]: pushed changes for this. needed to actually use the var :)

**Comment by beganovich** [comment]
Thank you for your contribution! :+1:

**Pull request merged**